streaming/src/main/resources/reference.conf

```diff
@@ -1,16 +1,5 @@
 gearpump {
 
-  serializers {
```

Comment:
This config is not exposed to user because it's in streaming's jar file. User still can config this in gear.conf